src/test/java/seedu/address/storage/JsonAdaptedRoomTest.java

```diff
@@ -1,17 +1,34 @@
 package seedu.address.storage;
 
 import static org.junit.jupiter.api.Assertions.assertEquals;
+import static seedu.address.testutil.Assert.assertThrows;
+import static seedu.address.testutil.TypicalRooms.ROOM_INVALID_PATIENT_PRESENT_OCCUPIED_FALSE;
 import static seedu.address.testutil.TypicalRooms.ROOM_NO_PATIENT_NO_TASK;
+import static seedu.address.testutil.TypicalRooms.ROOM_PATIENT_ALICE_NO_TASK;
 
 import org.junit.jupiter.api.Test;
 
+import seedu.address.commons.exceptions.IllegalValueException;
+
 class JsonAdaptedRoomTest {
 
     @Test
-    public void toModelType_returnsRoom() throws Exception {
+    public void toModelType_returnsEmptyRoom() throws Exception {
         JsonAdaptedRoom room1 = new JsonAdaptedRoom(ROOM_NO_PATIENT_NO_TASK);
-
         assertEquals(ROOM_NO_PATIENT_NO_TASK, room1.toModelType());
     }
 
+    @Test
+    public void toModelType_returnsOccupiedRoom() throws Exception {
```

Comment:
you can consider renaming your test method to `toModelType_returnsOccupiedRoom_success()` here.

Reply:
Okay